<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnCetak">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cetak</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Acara</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editnmAcara">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableAcara">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>441</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editIDAcara">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tanggal Acara</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpah</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nama Acara</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditAcara">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2025</year>
     <month>1</month>
     <day>13</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Deskripsi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editLokasi">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lokasi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editDeskrip">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dibuat Oleh</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editOleh">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
